frontend/playground-frontend/src/components/ClassicalMLModel/ClassicalMLModel.js

```diff
@@ -25,29 +25,83 @@ import {
   CSVInputFile,
   CSVInputURL,
 } from "../index";
+import { sendToBackend } from "../helper_functions/TalkWithBackend";
+import { toast } from "react-toastify";
+import DataTable from "react-data-table-component";
 
 const ClassicalMLModel = () => {
   const [customModelName, setCustomModelName] = useState(
     `Model ${new Date().toLocaleString()}`
   );
   const [addedLayers, setAddedLayers] = useState([]);
-  const [usingDefaultDataset, setUsingDefaultDataset] = useState(null);
+  const [usingDefaultDataset, setUsingDefaultDataset] = useState(
+    DEFAULT_DATASETS[0]
+  );
   const [shuffle, setShuffle] = useState(BOOL_OPTIONS[1]);
   const [email, setEmail] = useState("");
   const [dlpBackendResponse, setDLPBackendResponse] = useState();
   const [beginnerMode, setBeginnerMode] = useState(true);
   const [inputKey, setInputKey] = useState(0);
   const [testSize, setTestSize] = useState(0.2);
   const [problemType, setProblemType] = useState(PROBLEM_TYPES[0]);
+  const [csvDataInput, setCSVDataInput] = useState([]);
+  const [uploadedColumns, setUploadedColumns] = useState([]);
+  const [fileURL, setFileURL] = useState("");
+  const [, setOldCSVDataInput] = useState([]);
```

Comment:
is this a typo? @dwu359

Reply:
No, that's not a typo, that's was the only I could get typescript to stop complaining about an unused variable without using ts-ignore

Reply:
It's anti-pattern but it works, yes. It begs the question why you need that setState in the first place since you can't access the results

Reply:
But in this case, it's required by the smaller components so we can just leave this here